docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,459 @@
+Using Public Clouds as backends
+===============================
+
+Introduction
+------------
+
+As stated in our `GETTING STARTED guide <../GETTING_STARTED/#location-configuration>`__,
+new backends can be added by creating a region (also called location constraint)
+with the right endpoint and credentials.
+This section of the documentation shows you how to set up our currently
+supported public cloud backends:
+
+- `Amazon S3 <#aws-s3-as-a-backend>`__ ;
+- `Microsoft Azure <#microsoft-azure-as-a-backend>`__ .
+
+For all Public Cloud backends, you'll have to do some editing inside the
```

Comment:
For each public cloud backend, you will have to edit your CloudServer locationConfig.json and do a few setup steps on the applicable public cloud backend.

Reply:
changed